### ISSUE TITLE
[iOS] - rider create profile screen updates

### ISSUE THREAD

**Issue by phuong37** [trietphamvm/DAR#27]
minor changes - can update anytime :)
-update Photo Galary to Photo Gallery
-increase height of form fields
-change add photo to Add Photo + and increase font size to the same size as form field text. please also increase hotspot to make the Add Photo + link clickable.

![3_dar_userinfo](https://cloud.githubusercontent.com/assets/14860705/10156028/2d232b1a-6631-11e5-9ff2-fee681f3a6b9.png)

**Comment by phucdoan** [comment]
-when user types in the field, please remove the field name and replace with what the user is typing (remove first name, last name, etc.) => call by placeholder

=> Actually, user need to known which info they was fill on time. So placeholder still alive to remind them. But if it no need, i'll make them go away :)

**Comment by nguyendtt** [comment]
I actually agree with @phucdoan on the placeholders/field names @phuong37. I think it's a nice reminder as sometimes I type the wrong info in the wrong field. If you feel strongly about removing the field names, let's discuss further.

**Comment by phuong37** [comment]
hi anh @phucdoan - it's a very good point. can we have it disappear if the user types something but if they don't click in the box, show the first name, last name, etc.? i think that will help.

**Comment by trietphamvm** [comment]
@phuong37, actually this is a new style of placeholder and I suggested @phucdoan to keep it. It’s cleared and nice.

Thanks
TP

**Comment by phuong37** [comment]
chi @nguyendtt it's very common practice to remove when a user actually types something. please refer lyft and uber app. it's a good balance between usability and functionality. if we always include the title of the field, we will use up valuable space in the app, especially with the driver registration which is already very long. thoughts?

**Comment by phuong37** [comment]
anh @trietphamvm - it is a very nice feature but i just wonder if it's necessary? let me discuss with the design team. from a usability perspective, it is considered common practice for web and app forms to not have the header.

**Comment by trietphamvm** [comment]
Actually it’s good to have, right @phuong37? So I suggest to keep it. Let discuss with the team anyway.

Thanks
TP

**Comment by phuong37** [comment]
anh @trietphamvm - if it's not a usability issue, please keep the intent of the design as is. the title should not disappear until the user actually starts typing which makes it intuitive enough. this is not a matter of preference, it is standard practice now.

**Comment by trietphamvm** [comment]
@phuong37, it’s actually new style of input for software nowadays, it really helpful when we have a lot of data for input, if you want to review again, I’m sure you won’t know which information you did input without that placeholder still there. Trust me, let try this time. it won’t be redundant.

You know it was very difficult to implement in the past. In case it make trouble for user, we can remove later. Let wait for user feedback on this Phuong.

Thanks
TP

**Comment by tle960** [comment]
Hi everyone,

I noticed that we're having a lively discussion about the input form
fields.  Here's my take as the designer: I like the reminder popping up as
the user is typing to help guide him/her, however, I do not like that it
stays there after input.  That's adding more data then is necessary and
makes everything clustered with unnecessary info.  I don't need to be
reminded that my first name is Thanh after I have inputed it.  I believe
only the inputed information should stay after completion.

Thanh

**Comment by trietphamvm** [comment]
yes @tle960, but in case you have some fields for input. That’s ok. How do you think if you have a lot of fields and then you want to review all of them after input?

I have made a lot of projects, at the beginning, placeholder is not existed, the people place the label on the top of the input. Then when the technology improved, the placeholder is generated and I think nowadays, the placeholder will be changed like that.

We should consider this behavior as end-user, not a developer or designer. 

Anyway, please discuss with the team there and listen for the people is more end-user, I will make you happy if the team approve :)

This is a good point for discussion anyway.

Thanks
TP

**Comment by phuong37** [comment]
anh @trietphamvm, it is the end-user we have in mind when we design. :) please keep in mind that we put a lot of thought into keeping the interface clean, simple and intuitive, and we also research competitor and leading industry apps to help inform us. even with our longest form, the fields are self-explanatory and are broken out into sections that are labeled already. although we feel it is unnecessary to include titles, at the very minimum, we need to make some design adjustments if we are going to keep them. @tle960 please upload an updated design to accommodate for these changes. thanks all!

**Comment by nguyendtt** [comment]
@phuong37  I can't tell if the form field height has been updated, would you please verify.
@phucdoan There are still typos/mispellings on the "Select an Image" or "Select an image to change your avatar" popup.  They should be "Photo Gallery" and "Take a Picture".  Currently they read "Photo Gallary" and "Take a picture"

![img_3275](https://github-cloud.s3.amazonaws.com/assets/14857908/10565229/eb0946da-7588-11e5-8c56-cb22977a81c9.PNG)
![img_3280](https://github-cloud.s3.amazonaws.com/assets/14857908/10565240/362e2054-7589-11e5-83a7-88085109179a.PNG)
